synchro/__init__.py

```diff
@@ -768,13 +768,27 @@ def __init__(self, kms_providers, key_vault_namespace, key_vault_client=None, **
 class ClientEncryption(Synchro):
     __delegate_class__ = motor.MotorClientEncryption
 
-    def __init__(self, kms_providers, key_vault_namespace, key_vault_client, codec_options):
+    def __init__(
+        self,
+        kms_providers,
+        key_vault_namespace,
+        key_vault_client,
+        codec_options,
+        kms_tls_options=None,
+    ):
         self.delegate = motor.MotorClientEncryption(
-            kms_providers, key_vault_namespace, key_vault_client.delegate, codec_options
+            kms_providers,
+            key_vault_namespace,
+            key_vault_client.delegate,
+            codec_options,
+            kms_tls_options,
```

Comment:
This needs to be kms_tls_options=kms_tls_options otherwise we're implicitly passing io_loop=kms_tls_options.